locales/en/index.yml

```diff
@@ -3154,6 +3154,10 @@ features:
           issuedDate: "Valida dal"
           expirationDate: "Valida fino"
           restrictionConditions: "Codici"
+      unrecognizedData:
+        title: "Non riconosci alcuni dati?"
+        body: "Se ci sono errori nei dati o vuoi capire meglio cosa significano, puoi contattare il Ministero dell’Interno."
+        cta: "Chiedi informazioni o segnala errori"
```

Comment:
Is this change intended?

Reply:
Nope, it is a change that comes up after every merge of the base branch 😅

Reply:
Removed here https://github.com/pagopa/io-app/pull/6163/commits/03f76f640138c28953f263dbf02ca3670e88d49f